stoobly_agent/app/cli/intercept_cli.py

```diff
@@ -136,6 +146,22 @@ def configure(**kwargs):
 
         print(f"Updating {_mode} policy to {kwargs['policy']}")
 
+    if kwargs['strategy']:
+        active_mode = settings.proxy.intercept.mode
+
+        if active_mode == mode.RECORD:
```

Comment:
Since `test_strategy` options are available if active_mode is `mode.TEST`, we should allow updating test strategy as well.